web/src/api/exceptions/errorMessages.ts

```diff
@@ -0,0 +1,19 @@
+export const REQUIRED = (friendlyName: string) =>
```

Comment:
Can we capitalize the name of this file. It's a file with just constants, so I think I like PascalCase.